.github/workflows/integration-test-dumpling-common.yml

```diff
@@ -0,0 +1,56 @@
+name: DumplingCommon
+on:
+  workflow_call:
+    inputs:
+      debug:
+        type: boolean
+        description: 'set tmate on failure'
+        required: true
+      mysql_version:
+        type: string
+        description: 'specify mysql version'
+        required: true
+
+jobs:
+  integration-test:
+    runs-on: ubuntu-latest
+    timeout-minutes: 15
+    strategy:
+      fail-fast: true
+    services:
+      mysql:
+        image: mysql:${{ inputs.mysql_version }}
+        env:
+          MYSQL_ALLOW_EMPTY_PASSWORD: yes
+        ports:
+          - 3306:3306
+        options: --health-cmd="mysqladmin ping" --health-interval=10s --health-timeout=5s --health-retries=3
+    steps:
+      - name: 'checkout repository'
+        uses: actions/checkout@v3
+      - name: 'set up golang'
+        uses: actions/setup-go@v3
+        with:
+          go-version: 1.18
+      - name: 'try to use build cache'
+        uses: actions/cache@v3
+        with:
+          path: |
+            ~/.cache/go-build
+            ~/go/pkg/mod
+            ~/Library/Caches/go-build
+            ~\AppData\Local\go-build
+          key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+          restore-keys: |
+            ${{ runner.os }}-go-
+      - name: 'download binary dependencies'
+        run: sh dumpling/install.sh
+      - name: 'build tidb'
+        run: make server
+      - name: 'build lightning'
```

Comment:
personnally prefer fresh lightning here

Reply:
I'll keep it. I though the original test is to test both dumpling and lightning. Anyway removing the build does not make it faster because of cache have cut about 4 minutes build time.